pycobertura/reporters.py

```diff
@@ -177,10 +177,41 @@ def __init__(self, *args, **kwargs):
         self.no_file_sources_message = kwargs.pop(
             "no_file_sources_message", "Rendering of source files was disabled."
         )
+        self.sort_by_uncovered_lines = kwargs.pop(
+            "sort_by_uncovered_lines", False
+        )
         super(HtmlReporter, self).__init__(*args, **kwargs)
 
+    def _sort_summary_lines(self, summary_lines):
```

Comment:
Could this method move under `Reporter` so it works for all types of reports? What do you think?